bodhi/server/services/releases.py

```diff
@@ -57,6 +58,10 @@
                    # Note, this 'rw' is not a typo.  the @comments service has
                    # a ``post`` section at the bottom.
                    cors_origins=bodhi.server.security.cors_origins_rw)
+critpath_packages = Service(name='critpath_packages', path='/releases/{name}/critpath-packages',
+                            description='Retrieve and edit critical path packages.',
+                            factory=security.AdminACLFactory,
```

Comment:
This makes it so that only authorized users can ```GET``` the critpath packages:

```
[vagrant@bodhi-dev bodhi]$ curl -I http://localhost:6543/releases/F28/critpath-packages
HTTP/1.1 403 Forbidden
Content-Length: 10512
Content-Type: text/html; charset=UTF-8
Date: Wed, 15 Aug 2018 16:26:58 GMT
Server: waitress
X-Content-Type-Options: nosniff
```

I *think* it is possible to move the ```factory``` argument down to the service handlers below to require admin for write calls but allow read calls to anyone.

Another way to do it would be to make a new factory that inspects the request to determine whether admin ACLs are required. Either way is fine with me, but we do want to allow open reading of the critpath packages while requiring admin permissions to write them. As it is, ```bodhi releases critpath list F28``` does not work.